### PR TITLE
Enhancement/76

### PR DESCRIPTION
**Pull request by BhosalePriyanka**
Please check

### CODE REVIEW

File: spectrum/templates/base.html [hunk 1]
```diff
@@ -68,6 +68,7 @@
 
     <nav class="bg-light text-center fixed-bottom p-1 mt-5">
       <span>Supported by <a href="https://www.spiritofchristmas.org.uk/">Spirit Of Christmas</a>&nbsp;&copy;&nbsp;(Charity #1185011)</span>
+      <br/>
```
Comment (nmunro) [inline spectrum/templates/base.html:71]: why is this here?

File: info/templates/info/event_detail.html [hunk 2]
```diff
@@ -38,13 +38,15 @@ <h4>Location:</h4>
         <p>{{ event.location.post_code }}</p>
       </div>
       <div>
-        <p>{{ event.contact }}</p>
+        <span><a href="tel:+4733378901">{{ event.contact.phone_number}}</a>
+        <a href="mailto:https://www.google.com">{{ event.contact.email}}</a></span>
```
Comment (nmunro) [inline info/templates/info/event_detail.html:42]: This is not correct.

The href does not reference the phone number or the email, why are they not being displayed to the user?

**Review by nmunro — CHANGES_REQUESTED** [review]

File: spectrum/templates/base.html (hunk 1 above)
Comment (BhosalePriyanka) [reply spectrum/templates/base.html]: forget to remove...removed now

File: info/templates/info/event_detail.html (hunk 2 above)
Comment (BhosalePriyanka) [reply info/templates/info/event_detail.html]: I kept blank href link now,  as I dont know how to invoke python code in it.

**Review by nmunro — APPROVED** [review]
